src/java.desktop/share/classes/java/applet/package-info.java

```diff
@@ -36,7 +36,11 @@
  * running applets. For example, the applet context could be a Web browser or an
  * applet development environment.
  * <p>
- * The APIs in this package are all deprecated without replacement.
+ * Deprecated.
+ * This package has been deprecated and may be removed in a future version of the Java Platform.
```

Comment:
That should be `@deprecated This package ...`. See [java/rmi/activation/package-info.java#L41](https://github.com/openjdk/jdk/blob/master/src/java.rmi/share/classes/java/rmi/activation/package-info.java#L41).

Reply:
The deprecation description should point to the new API which might be used instead of the deprecated ones. So the text "deprecated without replacement" was intentionally added, it will be good to preserve it.